docs/private-networks/tutorials/sirato-explorer.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Privacy Enabled Besu - Sirato (Epirus) Explorer
+description: Using the Sirato Explorer on a privacy enabled Besu
+---
+
+# Sirato Explorer
+
+Sirato is an EVM explorer that provides an overview over the whole network, such as block information, view contracts metadata as well as search for transactions. Sirato supports public and private networks. This tutorial will walk you through how to setup Sirato Explorer with privacy-enabled Besu network.
```

Comment:
I'd combine the find out more section at the bottom of the page here, something like:

```
Sirato is an EVM explorer that provides an overview over the whole network, and provides a bunch of additional functionality like block information, contracts metadata, transaction searches etc and an overview can be found [here](https://medium.com/web3labs/epirus-ethereum-saas-blockchain-explorer-d5d961717d15). Sirato supports public and private networks. Sirato free (used here) has some [feature limitations](insert link of what these are ) and you can find out more information about Sirato [here](https://www.web3labs.com/sirato)

This tutorial will walk you through how to setup Sirato Explorer with privacy-enabled Besu network.

```

Update the links but something like that perhaps?